app/templates/components/ilios-users.hbs

```diff
@@ -1,10 +1,11 @@
 <div class="filters">
   <div class="filter user-search">
-    {{one-way-search
-      value=query
-      update=setQuery
```

Comment:
Why is this action going away? If I'm reading `users.hbs` right setting the query also changes the URL so it needs to happen at a higher level.

Reply:
ok, i missed that URL changing part. taking it back to the workbench. thanks!